end_to_end/nlp_mlops_company_sentiment/02_nlp_company_earnings_analysis_pipeline.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #72.        ],  # dependencies on both Finbert and Pegasus Deployment steps
</pre><p>Some occurrences of <code>Finbert</code> /<code>FinBert</code> were changed to <code>FinBERT</code> in this PR - do a find-and-replace to ensure consistency?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3056/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>